src/View/StringTemplate.php

```diff
@@ -180,7 +180,7 @@ protected function _compileTemplates(array $templates = [])
             }
 
             $template = str_replace('%', '%%', $template);
-            preg_match_all('#\{\{([\w\d\._]+)\}\}#', $template, $matches);
+            preg_match_all('#\{\{([\w\._]+)\}\}#', $template, $matches);
```

Comment:
Do we have a test case for this to not introduce an accidental regression?

Reply:
One of the un-testable things: a greedy regex. `\d` is sub-set of `\w`, what only can hurt performance.